cmd/watch.go

```diff
@@ -88,6 +88,9 @@ func watch() {
 	ticker := time.NewTicker(5 * time.Second)
 	defer ticker.Stop()
 
+	fmt.Printf("Timeout is %d", timeout)
```

Comment:
Should use `logWithCommand` to log messages if we want to keep this debug message

Reply:
fixed